AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -1003,13 +1025,39 @@ def postfx_gradient(property: str = 'name') -> str:
         """
         PostFX Gradient Weight Modifier component properties. Requires PostFX Layer component.
           - 'requires' a list of component names as strings required by this component.
-            Use editor_entity_utils EditorEntity.add_components(list) to add this list of requirements.\n
+            Use editor_entity_utils EditorEntity.add_components(list) to add this list of requirements.
+          - 'Gradient Entity Id' a separate entity id containing a gradient component.
+          - 'Opacity' factor multiplied by the current gradient before mixing. (float 0.0 to 1.0)
+          - 'Invert Input' swap the gradient input order black/white behave oppositely (bool)
+          - 'Enable Levels' toggle the application of input/output levels (bool)
```

Comment:
Input and output comment lines (5 lines below this) exceed line length guidelines